### ISSUE TITLE
next update for Panther

### ISSUE THREAD

**Issue by RLovering** [geneontology/panther-enrichment#18]
Hi 
we are writing a paper and would like to use PANTHER for our enrichment analysis but the online version is using Annotation Version and Release Date: GO Ontology database Released 2019-02-02.

I assume this means both the annotation (gene association) and the ontology files?

Will new versions of the annotation (gene association) and the ontology files be included any time soon?

Thanks

Ruth

cc @barbaraczub, @thomaspd, @huaiyumi

**Comment by huaiyumi** [comment]
Yes, we are planning to have an update to the current version soon (within the next few weeks).

Thanks,

Huaiyu

**Comment by BarbaraCzub** [comment]
Hi Huaiyu,

This is great. Thank you for letting us know. I just wanted to followup on this to help us to plan the paper writing around our other activities.

Thanks,
Barbara

**Comment by BarbaraCzub** [comment]
Hi Huaiyu,

I noticed that Panther is now using the 2019-07-03 release of GO. Thank you so much for implementing this!

Going back to Ruth's original question, could you please confirm whether we understand correctly that this release date refers to both the annotation (gene association) and the ontology files?

Thank you,
Barbara

**Comment by BarbaraCzub** [comment]
Here is a fragment of my email conversation with Huaiyu with Huaiyu's answers to our questions:
 
Q.       Does the 'GO Ontology database released 2019-07-03' refer to both the release of the annotation (gene association) file and the ontology file? Or is it only the gene association file? Or only the ontology file?
A.        This is for both the ontology and the gene associations.
 
Q.       The Analysis Type 'PANTHER Overrepresentation Test' says that it was 'Released 20190711'. What does it mean that the PANTHER Overrepresentation Test was released on that date? What should the Analysis Type change with date?
A.        We made some minor tweaks on the tool. This is mainly the UI updates. The date reflects the date the code was updated.
